### DISCUSSION TITLE
Filebrowser with noauth

### DISCUSSION THREAD

**Discussion by di5cord20** [tteck/Proxmox#1861] [Help Needed] [answered]
Filebrowser is up and running on my pve node using the ```bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/filebrowser.sh)"``` script.  I'd like to disable authentication and tried following https://filebrowser.org/configuration/authentication-method#no-authentication by running ```filebrowser config set --auth.method=noauth``` but got a timeout response.  Any ideas how to remove auth?

**Comment by tteck** [comment, marked as answer]
Execute the command below in the LXC console
```
systemctl stop filebrowser.service
rm /root/filebrowser.db
filebrowser config init -a '0.0.0.0'
filebrowser config set -a '0.0.0.0'
filebrowser users add ID 1 --perm.admin
filebrowser config init --auth.method=noauth
filebrowser config set --auth.method=noauth
systemctl start filebrowser.service
```

**Reply by di5cord20** [reply]
Worked like a charm!  Thanks again @tteck for your help.

**Comment by mrbernnz** [comment]
Just in case someone has an issue with filebrowser error (i.e. The command could not be located because '/usr/local/bin' is not included in the PATH environment variable.)

You can try the full path `/usr/local/bin/filebrowser` instead.

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/blob/a9b44432c4ff5810582de9bc11c03e7c8e346d83/misc/filebrowser.sh#L93
